flext doesn't compile on ARM (Raspbian Buster)

I tried this on a Raspberry Pi4 with Raspbian Buster (which corresponds to Debian stable), gcc is "gcc (Raspbian 8.3.0-6+rpi1) 8.3.0". Here's the error message that I get:

~~~
g++ -c  -msse -mfpmath=sse -ffast-math -DFLEXT_USE_CMEM -O3  -I/usr/include/pd -pthread -fPIC -fvisibility-inlines-hidden -DFLEXT_USE_SIMD -DNDEBUG -DFLEXT_EXPORTS -DFLEXT_SYS=2 -DPD -I../../../../pd/src -I../../../../pd/include -I../../../../pd/include/pd -I/usr/include/stk -I/usr/include/SndObj source/flbase.cpp -o pd-linux/release-single/flbase.opp
g++: error: unrecognized command line option ‘-msse’; did you mean ‘-fdse’?
g++: error: unrecognized command line option ‘-mfpmath=sse’
make[6]: *** [buildsys/lnx/gnumake-gcc-flext.inc:25: pd-linux/release-single/flbase.opp] Error 1
~~~

Full buildlog here: 
[pd-l2ork-buildlog.txt](https://github.com/pd-l2ork/pd/files/3489934/pd-l2ork-buildlog.txt)

Looks like the flext configuration defaults to targeting x86. Is there an easy way to fix that? That is, without having to patch up the flext sources?

--- a/debuild/Makefile
+++ b/debuild/Makefile
@@ -63,7 +63,7 @@ debchange:
 
 debclean:
 	rm -rf $(debdist)
-	rm -f pd-l2ork_*+git*
+	rm -f pd-l2ork_*+git* pd-l2ork-dbgsym_*+git*
 
 deb: $(debsrc)
 # Unpack the source into our temporary build directory.

--- a/debuild/debian/rules
+++ b/debuild/debian/rules
@@ -4,13 +4,19 @@
 # Uncomment this to turn on verbose mode.
 #export DH_VERBOSE=1
 
+ifeq ("$(shell dpkg-architecture -qDEB_HOST_ARCH)","armhf")
+buildopt = -R
+else
+buildopt = -B
+endif
+
 %:
 	dh $@
 
 override_dh_auto_configure:
 
 override_dh_auto_build:
-	cd l2ork_addons && ./tar_em_up.sh -B -n
+	cd l2ork_addons && ./tar_em_up.sh $(buildopt) -n
 
 override_dh_auto_install:
 	mkdir -p debian/pd-l2ork && mv packages/linux_make/build/* debian/pd-l2ork

